sql/codegen.go

```diff
@@ -46,6 +40,30 @@ type filler struct {
 	connectionConfig
 }
 
+// TODO(tonyyang): This is currently a quick hack to map from SQL
+// field types to feature types.  We will enhance it to support more
+// complex cases like cross features.
+func newColumnType(n, t string) *columnType {
+	return &columnType{n, t}
+}
+
+func translateColumnToFeature(fts *fieldTypes, driverName, ident string) (*columnType, error) {
+	ct, ok := fts.get(ident)
+	if !ok {
+		return nil, fmt.Errorf("genTF: Cannot find type of field %s", ident)
+	}
+	ctype, e := universalizeColumnType(driverName, ct.Type)
+	if e != nil {
+		return nil, e
+	}
+
+	if ctype == "FLOAT" {
+		return &columnType{ident, "numeric_column"}, nil
+	}
+	return &columnType{ident, ""}, nil
```

Comment:
We should probably return an error if we can't map a `ctype` to a feature column, so that we can make error message like https://github.com/sql-machine-learning/sqlflow/issues/350 more informative.

Reply:
return error like "unsupported field type XXX"